scenarios/contributions-menu.json

```diff
@@ -175,13 +175,74 @@
       "onReadyScript": "premiums/delete-modal.js"
     },
     {
-      "label": "New Price Set",
+      "label": "Contributions - New Price Set",
       "url": "{url}/civicrm/admin/price?reset=1&action=add"
     },
     {
-      "label": "Manage Price Set",
+      "label": "Contributions - Manage Price Set",
       "url": "{url}/civicrm/admin/price?reset=1"
     },
+    {
+      "label": "Contributions - Manage Price Set - Add Price Fields",
+      "url": "{url}/civicrm/admin/price?reset=1",
+      "onReadyScript": "price-set/add-price-fields.js"
+    },
+    {
+      "label": "Contributions - Manage Price Set - Edit Price Fields Set - results ",
```

Comment:
why are we adding "- results" everywhere? Some of these scenarios have nothing to do with searches, or any action that would return actual results

Reply:
Yes that is right! Let me update the label